src/main/java/uk/gov/ons/ssdc/notifysvc/messaging/EmailRequestEnrichedReceiver.java

```diff
@@ -26,6 +26,12 @@ public class EmailRequestEnrichedReceiver {
   @Value("${email-request-enriched-delay}")
   private int emailRequestEnrichedDelay;
 
+  @Value("${errorhandling.rate-limit-error-http-status}")
+  private int rateLimitErrorHttpStatus;
+
+  @Value("${errorhandling.rate-limiter-exception-message}")
+  private String rateLimiterExceptionMessage;
+
```

Comment:
Why are these values passed in instead of being a constant or something similar?